server/db/analysisReports.js

```diff
@@ -0,0 +1,88 @@
+const AnalysisReport = require('./model/AnalysisReport')
+
+const randomNumbers = (size) => Math.random().toString().substring(2, size + 2)
+
+const generateUuid = async (db) => {
+  const uuid = `${randomNumbers(4)}-${randomNumbers(4)}-${randomNumbers(2)}`
```

Comment:
Not sure this is random enough. It could probably be brute forced quite easily.

Reply:
Agree, will replace it for a CSPRNG generator